code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -82,16 +82,16 @@
 	desc = "A floating, fragile spore."
 	icon_state = "blobpod"
 	icon_living = "blobpod"
-	health = 40
-	maxHealth = 40
+	health = 30
+	maxHealth = 30
 	verb_say = "psychically pulses"
 	verb_ask = "psychically probes"
 	verb_exclaim = "psychically yells"
 	verb_yell = "psychically screams"
 	melee_damage_lower = 2
 	melee_damage_upper = 4
 	obj_damage = 20
-	environment_smash = 0
+	environment_smash = 1
```

Comment:
Oh, I see. Makes sense. Although it'll be weird for blob sport to do the big impact noise.